utilities/storage.py

```diff
@@ -606,12 +607,22 @@ def get_test_artifact_server_url(schema="https"):
     """
     artifactory_connection_url = py_config["servers"]["https_server"]
     LOGGER.info(f"Testing connectivity to {artifactory_connection_url} {schema.upper()} server")
-    resp = requests.get(artifactory_connection_url, headers=utilities.infra.get_artifactory_header(), verify=False)
-    assert resp.status_code == requests.codes.ok, (
-        f"Unable to connect to test image server: {artifactory_connection_url} "
-        f"{schema.upper()}, with error code: {resp.status_code}, error: {resp.text}"
-    )
-    return py_config["servers"][f"{schema}_server"]
+    try:
```

Comment:
Please define `sample`, otherwise it will raise an error in the LOGGER.error
```suggestion
    sample = None
    try:
```

Reply:
+1

Reply:
done